examples/calib_udelay/Kconfig

```diff
@@ -6,7 +6,6 @@
 config EXAMPLES_CALIB_UDELAY
 	tristate "Calibration tool for udelay"
 	default n
-	select CLOCK_MONOTONIC
```

Comment:
There was no change to remove CLOCK_MONOTONIC from calib_udelay_main.c, so this breaks the build.

Reply:
CLOCK_MONOTONIC always exist after https://github.com/apache/incubator-nuttx/pull/5503, so CLOCK_MONOTONIC can be used without any precondition.

Reply:
Thanks for the quick response. My nuttx and apps git trees were out-of-sync and syncing them to the same place fixed this.